internal/usage/usage.go

```diff
@@ -222,35 +222,35 @@ dbName and collection are required only for built-in roles.`
+	LogRedacted                               = "Flag that indicates whether to replace emails, hostnames, IP addresses, and namespaces in API responses involving this job with random string values."
+	ServerUsageSkipRedacted                   = "Flag that indicates whether to skip the redaction of sensitive values. If you set this flag, Ops Manager logs the original values for hostnames, IP addresses, and namespaces in API responses for this job."
+	Sparse                                    = "Flag that creates a sparse index. To learn more, see https://dochub.mongodb.org/core/index-sparse-manual."
 	Locale                                    = "Locale that the ICU defines."
 	CaseLevel                                 = "Flag that enables index case comparison. This flag applies only if the strength level is set to 1 or 2."
-	CaseFirst                                 = "Flag that, if specified, determines the sort order of case differences during tertiary level comparisons."
+	CaseFirst                                 = "Flag that determines the sort order of case differences during tertiary level comparisons."
 	Strength                                  = "Level of comparison to perform."
-	Alternate                                 = "Flag that, if specified, determines whether collation should consider whitespace and punctuation as base characters during comparisons."
-	MaxVariable                               = "Flag that, if specified, determines which characters to ignore. This flag applies only if indexConfigs.collation.alternate is set to shifted."
-	NumericOrdering                           = "Flag that, if set to true, indicates that collation compares numeric strings as numbers. If set to false, collation compares numeric strings as strings."
-	Normalization                             = "Flag that, if set to true, collation checks if text requires normalization and performs normalization to compare text."
-	Backwards                                 = "Flag that, if set to true, strings with diacritics sort from the back to the front of the string."
+	Alternate                                 = "Flag that determines whether collation should consider whitespace and punctuation as base characters during comparisons."
+	MaxVariable                               = "Flag that determines which characters to ignore. This flag applies only if you set indexConfigs.collation.alternate to shifted."
+	NumericOrdering                           = "Flag that indicates that collation compares numeric strings as numbers. If you set to false, collation compares numeric strings as strings."
+	Normalization                             = "Flag that indicates whether collation checks if text requires normalization and performs normalization to compare text."
+	Backwards                                 = "Flag that indicates whether strings with diacritics sort from the back to the front of the string."
```

Comment:
**Question:**

@dianchenghu and @blva, are all booleans passed as simple flags, or do users actually have to specify "true" and "false"? See @sarahsimpers comment. If so, can we have a consistent behavior.

Reply:
any flag marked with `BoolVar` will be just passed as flags indeed, nobody needs to specify true, but some will have default to "true" and some will have default to "false"

Reply:
@corryroot  We set default values for each bool flag in code if they're not specified.
For each of the above under index create, the default is false.
So "indicates that" would probably be better for them.